### ISSUE TITLE
Quadrupole moment for 2H ? 

### ISSUE THREAD

**Issue by jkshenton** [lmmentel/mendeleev#100]
Hi again!

I can't find the quadrupole moment for deuterium. 

I see that it's not there in the https://www-nds.iaea.org/nuclearmoments/elec_mom_recomm.csv csv file, but it does appear in the web GUI of https://www-nds.iaea.org/nuclearmoments/index.html

It does also feature in the pdf report:
https://www-nds.iaea.org/publications/indc/indc-nds-0833.pdf

Perhaps we need to re-build the csv files for the quadrupole moments and the nuclear dipole magnetic moments from the latest  pdf reports?

**Comment by lmmentel** [comment]
Yup, I think there's some mismatch between the csv, web ui and the paper. Not sure yet how to resolve it.

**Comment by lmmentel** [comment]
Hey @jkshenton did you find a good solution by any chance?
